src/main/java/io/openepcis/convert/Conversion.java

```diff
@@ -0,0 +1,90 @@
+package io.openepcis.convert;
+
+import io.openepcis.constants.EPCISFormat;
+import io.openepcis.constants.EPCISVersion;
+import lombok.Getter;
+
+@Getter
```

Comment:
I don't like the get pattern for this kind of pattern.
use fromMediaType(), toMediaType(), etc.

 get method names somehow imply that set is available or was missed for some strange reason